docs/sources/articles/baseimages.md

```diff
@@ -41,15 +41,17 @@ GitHub Repo:
  - [Debian / Ubuntu](
    https://github.com/docker/docker/blob/master/contrib/mkimage-debootstrap.sh)
 
-## Creating a simple base image using `scratch`
+## Creating a simple base image using 'scratch'
```

Comment:
Unless the `TagStore` is defined elsewhere in the documentation, I'd rather not introduce that concept.

> The scratch image was created from a empty tar file as follows ...

Well, it *was* originally made this way but now it's not even an image layer at all. It's a placeholder name which signals to the build system that you want the next command in the DockerFile to be the first filesystem layer in the image. Note that this is distinct from what it used to be: an empty filesystem layer.